support/routes/v2/isobot/ops/botService.ts

```diff
@@ -0,0 +1,97 @@
+import { createHmac, timingSafeEqual } from "node:crypto"
+
+import qs from "qs"
+
+import { config } from "@config/config"
+
+import logger from "@root/logger/logger"
+import WhitelistService from "@root/services/identity/WhitelistService"
+
+export interface SlackPayload {
+  token: string
+  team_id: string
+  team_domain: string
+  channel_id: string
+  channel_name: string
+  user_id: string
+  user_name: string
+  command: string
+  text: string
+}
+
+class BotService {
+  whitelistService: WhitelistService
+
+  constructor(whitelistService: WhitelistService) {
+    this.whitelistService = whitelistService
+  }
+
+  public verifySignature(
```

Comment:
@spaceraccoon  - could i just trouble you to check this? not entirely sure on this and is definitely not my area of expertise HAHA

Reply:
resolving, done in #1391